common/source/docs/common-escs-and-motors.rst

```diff
@@ -29,3 +29,4 @@ ArduPilot supports a wide variety of ESCs and motors.  The pages below provide s
     Trolling motors <trolling-motor>
 [/site]
     UAVCAN ESCs <common-uavcan-escs>
+    Velocity ESCs <common-velocity-can-escs>
```

Comment:
if you can remove this link, I will add it on the future wiki changes page to link it to the main page, which I will merge into the wiki with a link in the TOC tree for Upcoming Features, since this is only supported in master and not stable....